app/controllers/reports/export_raw_reports_controller.rb

```diff
@@ -24,14 +17,8 @@ def raw_report
       )
     end
 
-    def generate_report_data_csv
-      ExportRawReportMailer.delay.raw_report_email(email_to_address, raw_report)
-      if request.xhr?
-        render nothing: true
-      else
-        flash[:notice] = I18n.t("controllers.reports.mail_queued", email: email_to_address)
-        redirect_to send("#{action_name}_facility_general_reports_path", current_facility)
-      end
+    def success_redirect_path
+      public_send("#{action_name}_facility_general_reports_path", current_facility)
```

Comment:
Looks like this was actually wrong on master. I'm getting `undefined method export_all_facility_general_reports_path'` if I turn of JS.  `action_name` will always be `export_all` in this case, so this would probably cause infinite redirects.

Doesn't really matter that much because it should always be getting hit via ajax.